WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -835,6 +835,11 @@ - (void)listenLocalNotifications
                            selector:@selector(handleLowMemoryWarningNote:)
                                name:UIApplicationDidReceiveMemoryWarningNotification
                              object:nil];
+    
+    [notificationCenter addObserver:self
+                           selector:@selector(handleSignInDidFinish:)
+                               name:SigninHelpers.WPSigninDidFinishNotification
```

Comment:
Because the account changing doesn't guarantee that the blogs are synced, which is when we get the default blog for the account (which was what was happening)